cmd/cluster-init/update_sanitize_prow_jobs_test.go

```diff
@@ -0,0 +1,111 @@
+package main
+
+import (
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+
+	"github.com/openshift/ci-tools/pkg/api"
+	"github.com/openshift/ci-tools/pkg/dispatcher"
+)
+
+func TestUpdateSanitizeProwJobs(t *testing.T) {
```

Comment:
I think we get the best balance of unit and integration tests by having one large integration test for "the whole tool" that runs the entrypoint and asserts about the state of the entire output directory. Think about a flow that:

1. copies some static testdata input to a temporary directory
2. runs the whole tool you're working on, on this new directory
3. uses `testutil.CompareWithFixtureDir` to compare the modified directory with a static testdata output directory

Reply:
Completely agree with you here. The output checking unit tests along with a big integration test would be overkill so I'm going to remove those in favor of the integration test.